api/pkg/apis/projectcalico/v3/felixconfig.go

```diff
@@ -611,6 +611,40 @@ type FelixConfigurationSpec struct {
 	// [Default: Auto]
 	BPFConntrackCleanupMode *BPFConntrackMode `json:"bpfConntrackMode,omitempty" validate:"omitempty,oneof=Auto Userspace BPFProgram"`
 
+	// BPFConntrackTimers overides the default values for the specified conntrack timer if
+	// set. Each value can be either a duration or `auto` to pick the value from
+	// a Linux conntrack timeout.
+	//
+	// Configurable timers are: CreationGracePeriod, TCPSynSent,
+	// TCPEstablished, TCPFinsSeen, TCPResetSeen, UDPLastSeen, GenericIPLastSeen,
+	// ICMPLastSeen.
+	//
+	// Unset values are replaced by the default values with a warning log for
+	// incorrect values.
+	//
+	// Current auto mappings:
+	//
+	// TCPSynSent: nf_conntrack_tcp_timeout_syn_sent
+	// TCPEstablished:    nf_conntrack_tcp_timeout_established
+	// TCPFinsSeen:       nf_conntrack_tcp_timeout_time_wait
+	// GenericIPLastSeen: nf_conntrack_generic_timeout
+	// ICMPLastSeen:      nf_conntrack_icmp_timeout
+	//
+	// If there is no mapping, 'auto' is replaced by the default value.
```

Comment:
```suggestion
```

I think all of this is better expressed as field-level comments, as each field has its own behavior. See my suggested format for the per-field comments below.